Code/GraphMol/MolPickler.cpp

```diff
@@ -1793,12 +1817,21 @@ Bond *MolPickler::_addBondFromPickle(std::istream &ss, ROMol *mol, int version,
 
     if (version < 7000) {
       streamRead(ss, tmpChar, version);
+      if (tmpChar < 0) {
```

Comment:
This is used often enough, I might have made a helper func

```
void streamReadChar(...) {
      if (tmpChar < 0) {
         throw MolPicklerException("invalid value in pickle");
       }
}
```

Reply:
yep